build/signedbuild.yml

```diff
@@ -310,6 +311,16 @@ jobs:
       uploadRoslyn: false
       uploadTSLint: false
 
+  - task: CopyFiles@2
+    displayName: 'Copy Files for Signing Validation'
+    inputs:
+      SourceFolder: '$(Build.SourcesDirectory)\src'
+      Contents: |
+       AccessibilityInsights\bin\Release\net472\**\?(*.exe|*.dll)
+       AccessibilityInsights.VersionSwitcher\bin\Release\net472\**\?(*.exe|AccessibilityInsights.*.dll)
```

Comment:
Any thoughts on ways that we might be able to future-proof this in case VersionSwitcher somehow picked up additional 3rd party assemblies?

Reply:
My only issue with the exclude approach is that if we add a new external dependency in the same manner as the existing dependencies (ie, one that is signed in the AccessibilityInsights project), we would then have to update the exclude list. 

That being said, if the situation we want to avoid is one where we inadvertently ship an unsigned dependency, the exclude approach will ensure that the signing validation task still catches the issue. I'll update the PR 🙂